### DISCUSSION TITLE
terraaform plan error.

### DISCUSSION THREAD

**Discussion by ephantusGachomba** [chiche-ds/30-Day-Terraform-challenge-#88] [General]
am getting this error when running terraform plan,
![terraform_plan_error](https://github.com/user-attachments/assets/d9e82f01-65af-488d-bdcd-11681e801e49)

**Comment by nadeemtaj371** [comment]
it seems like you don't provide, cloud provider details in your .tf files or you're not on the same directory. check this, try again

**Reply by ephantusGachomba** [reply]
I redid by running

aws configure

**Comment by kevshakes** [comment]
I see you are getting an invalid region error maybe you can share a screenshot of your main.tf file with emphasis on the AWS Provider. The issue might be you may have mistyped the AWS regions!
